lib/kaffe/group_member/worker/worker_supervisor.ex

```diff
@@ -0,0 +1,35 @@
+defmodule Kaffe.WorkerSupervisor do
+  @moduledoc """
+  Supervise the individual workers.
+  """
+
+  use Supervisor
+
+  require Logger
+
+  def start_link do
+    Supervisor.start_link(__MODULE__, :ok)
+  end
+
+  def start_worker_manager(pid) do
+    Supervisor.start_child(pid, worker(Kaffe.WorkerManager, []))
+  end
+
+  def start_worker(pid, message_handler, partition) do
+    Logger.debug "Starting worker for partition: #{partition}"
+    Supervisor.start_child(pid,
+      worker(Kaffe.Worker, [message_handler, partition], id: partition))
+  end
+
+  def init(:ok) do
+    Logger.info "event#startup=#{inspect self()}"
+
+    children = [
+    ]
```

Comment:
I wanted it to use `one_for_all` because I want all workers to be restarted if one of them crashes.

I think a `simple_one_for_one` might work, I wanted to think more about what would happen during the crash of a single worker and how it might recover.